src/main/java/org/opentripplanner/model/plan/WalkStep.java

```diff
@@ -124,14 +124,14 @@ public Optional<AbsoluteDirection> getAbsoluteDirection() {
   /**
    * When exiting a highway or traffic circle, the exit name/number.
    */
-  public String getExit() {
+  public String isExit() {
```

Comment:
Keep `getExit` (or rename to `getHighwayExitOrTrafficCirckeName()`), this brekes the JavaBean standard and is confusing.

Reply:
Oh, you're right, that was an oversight: 957783d333d357ae7191c0458063d6ce00eb41bb